phpstan.neon.dist

```diff
@@ -26,6 +26,9 @@ parameters:
 		- PERFLAB_OBJECT_CACHE_DROPIN_VERSION
 	strictRules:
 		strictArrayFilter: false
+	featureToggles:
+		# Ignore errors related to the plugin not running from inside a WP environment, so paths like ABSPATH . WPINC don't resolve.
+		requireFileExists: false
```

Comment:
Without this PHPStan complains about a couple dozen `require_once` calls:

<details><summary>Path in require_once() "..."  is not a file or it does not exist.    </summary>

```
------ ----------------------------------------------------------------------- 
  Line   plugins/optimization-detective/tests/test-optimization.php             
 ------ ----------------------------------------------------------------------- 
  212    Path in require_once() "./wp-includes/class-wp-customize-manager.php"  
         is not a file or it does not exist.                                    
 ------ ----------------------------------------------------------------------- 

 ------ ----------------------------------------------------------------------- 
  Line   plugins/performance-lab/includes/admin/load.php                        
 ------ ----------------------------------------------------------------------- 
  259    Path in require_once() "./wp-admin/includes/plugin.php" is not a file  
         or it does not exist.                                                  
  260    Path in require_once() "./wp-admin/includes/plugin-install.php" is     
         not a file or it does not exist.                                       
  261    Path in require_once() "./wp-admin/includes/class-wp-upgrader.php" is  
         not a file or it does not exist.                                       
  262    Path in require_once()                                                 
         "./wp-admin/includes/class-wp-ajax-upgrader-skin.php" is not a file    
         or it does not exist.                                                  
 ------ ----------------------------------------------------------------------- 

 ------ ----------------------------------------------------------------------- 
  Line   plugins/performance-lab/includes/admin/plugins.php                     
 ------ ----------------------------------------------------------------------- 
  95     Path in require_once() "./wp-admin/includes/plugin-install.php" is     
         not a file or it does not exist.                                       
  96     Path in require_once() "./wp-admin/includes/plugin.php" is not a file  
         or it does not exist.                                                  
 ------ ----------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------------------------- 
  Line   plugins/performance-lab/includes/server-timing/object-cache.copy.php                    
 ------ ---------------------------------------------------------------------------------------- 
  68     Path in require_once()                                                                  
         ".//performance-lab/includes/server-timing/class-perflab-server-timing-metric.php"      
         is not a file or it does not exist.                                                     
  68     Path in require_once()                                                                  
         ".//performance/includes/server-timing/class-perflab-server-timing-metric.php"          
         is not a file or it does not exist.                                                     
  68     Path in require_once()                                                                  
         ".//plugins/performance/includes/server-timing/class-perflab-server-timing-metric.php"  
         is not a file or it does not exist.                                                     
  69     Path in require_once()                                                                  
         ".//performance-lab/includes/server-timing/class-perflab-server-timing.php"             
         is not a file or it does not exist.                                                     
  69     Path in require_once()                                                                  
         ".//performance/includes/server-timing/class-perflab-server-timing.php"                 
         is not a file or it does not exist.                                                     
  69     Path in require_once()                                                                  
         ".//plugins/performance/includes/server-timing/class-perflab-server-timing.php"         
         is not a file or it does not exist.                                                     
  70     Path in require_once()                                                                  
         ".//performance-lab/includes/server-timing/load.php" is not a file or                   
         it does not exist.                                                                      
  70     Path in require_once()                                                                  
         ".//performance/includes/server-timing/load.php" is not a file or it                    
         does not exist.                                                                         
  70     Path in require_once()                                                                  
         ".//plugins/performance/includes/server-timing/load.php" is not a                       
         file or it does not exist.                                                              
  71     Path in require_once()                                                                  
         ".//performance-lab/includes/server-timing/defaults.php" is not a                       
         file or it does not exist.                                                              
  71     Path in require_once()                                                                  
         ".//performance/includes/server-timing/defaults.php" is not a file or                   
         it does not exist.                                                                      
  71     Path in require_once()                                                                  
         ".//plugins/performance/includes/server-timing/defaults.php" is not a                   
         file or it does not exist.                                                              
  85     Path in require_once() ".//object-cache-plst-orig.php" is not a file                    
         or it does not exist.                                                                   
 ------ ---------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------- 
  Line   plugins/performance-lab/tests/data/class-wp-filesystem-mockfilesystem.php  
 ------ --------------------------------------------------------------------------- 
  12     Path in require_once()                                                     
         "./wp-admin/includes/class-wp-filesystem-base.php" is not a file or        
         it does not exist.                                                         
 ------ --------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------- 
  Line   plugins/webp-uploads/tests/data/class-wp-image-doesnt-support-modern-images.php  
 ------ --------------------------------------------------------------------------------- 
  10     Path in require_once() "./wp-includes/class-wp-image-editor.php" is              
         not a file or it does not exist.                                                 
  11     Path in require_once()                                                           
         "./wp-includes/class-wp-image-editor-imagick.php" is not a file or it            
         does not exist.                                                                  
 ------ --------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------- 
  Line   plugins/webp-uploads/tests/data/class-wp-image-edit.php               
 ------ ---------------------------------------------------------------------- 
  1[66](https://github.com/WordPress/performance/actions/runs/10782751070/job/29903342323#step:10:67)    Path in include_once() "./wp-admin/includes/image-edit.php" is not a  
         file or it does not exist.                                            
 ------ ----------------------------------------------------------------------
```

</details> 

This is a new feature that landed in [1.12.1](https://github.com/phpstan/phpstan/releases/tag/1.12.1) last week.

I didn't actually find the `featureToggles` config documented anywhere. I found it in PHPStan's own config files:

https://github.com/phpstan/phpstan-src/blob/a50b75a81e41de985ffd5b5334319c471c9b9a33/conf/bleedingEdge.neon#L65

Adding this to our config file here works, however.

cc @thelovekesh